schemas/iso19139/src/main/plugin/iso19139/present/csw/iso-summary.xsl

```diff
@@ -161,12 +168,17 @@
 	<!-- =================================================================== -->
 
 	<xsl:template match="srv:SV_ServiceIdentification|
```

Comment:
This is for compliance with CSW, add missing (empty) tags and fill out the complex citation field instead of simply copying the first citation value (tile)